executor/show_test.go

```diff
@@ -462,6 +462,31 @@ func (s *testSuite) TestShowVisibility(c *C) {
 	tk.MustExec("drop database showdatabase")
 }
 
+// ShowDatabases show the first database is INFORMATION_SCHEMA
+func (s *testSuite) TestMoveInfoSchemaToFront(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustQuery("show databases").Check(testkit.Rows("INFORMATION_SCHEMA"))
+	tk.MustExec(`create user 'show'@'%'`)
```

Comment:
We can not use `show databases` without this in test?

Reply:
Why do we need to create a new user?

Reply:
Oh, I have deleted the cmd.